src/meta/src/barrier/command.rs

```diff
@@ -76,7 +76,7 @@ pub struct Reschedule {
 /// [`Command`] is the action of [`crate::barrier::GlobalBarrierManager`]. For different commands,
 /// we'll build different barriers to send, and may do different stuffs after the barrier is
 /// collected.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, strum::Display)]
```

Comment:
Nits: the inner fields of some enum item could be extremely large, I'm not sure if we should pick and only display some important info manually only for them. It's acceptable for me to keep it this way since only latest 10 events will be recorded.

Reply:
strum::Display only include the name. See [example here](https://docs.rs/strum/latest/strum/derive.ToString.html).